pkgs/development/python-modules/sphinxcontrib-trio/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-trio";
+  version = "1.1.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-nxupwdWWW1NOhSWNi2d92U6bGpoukYuFzNQlkFlrR8A=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [ sphinx ];
+
```

```suggestion
  nativeCheckInputs = [
  	pytestCheckHook
  ];
```
To ensure the tests are run. If the tests are not part of the pypi package switch the src to the Github repository.

---

You can disable that test using `disabledTests`. Probably `disabledTests = [ "test_end_to_end" ]` would work

---

It's added now, waiting for the CI to pass